src/transformation/visitors/literal.ts

```diff
@@ -62,78 +63,99 @@ const transformNumericLiteralExpression: FunctionVisitor<ts.NumericLiteral> = ex
     return lua.createNumericLiteral(Number(expression.text), expression);
 };
 
-const transformObjectLiteralExpression: FunctionVisitor<ts.ObjectLiteralExpression> = (expression, context) => {
-    const violations = findMultiAssignmentViolations(context, expression);
-    if (violations.length > 0) {
-        context.diagnostics.push(...violations.map(e => invalidMultiFunctionUse(e)));
-        return lua.createNilLiteral(expression);
-    }
-
-    let properties: lua.TableFieldExpression[] = [];
-    const tableExpressions: lua.Expression[] = [];
+const transformObjectLiteralExpressionOrJsxAttributes: FunctionVisitor<ts.ObjectLiteralExpression | ts.JsxAttributes> =
```

Comment:
The git diff here really got confused due to just one missing newline. Is this worth changing?

Reply:
Seems like this is prettier adding another indent due to the formatting after the first line became so much longer. I don't really mind as long as Prettier is happy, but it might be nicer not to touch all of this code for our git history/blames. You could try to somehow get prettier to format it differently, or maybe by aliasing the type for example.

Like I said, as long as prettier is happy I don't really mind.

Reply:
I think I'll leave it as is, since I don't want to fight with prettier.